emission/net/ext_service/habitica/proxy.py

```diff
@@ -26,8 +27,11 @@ def habiticaRegister(username, email, password, our_uuid):
   #want to access their Habitica account from the browser
   #Need to create a way from them to retrieve username/password
   habitica_user_table = edb.get_habitica_db()
-  habitica_user_table.insert({'user_id': our_uuid, 'habitica_username': username, 
-    'habitica_password': password, 'habitica_id': user_dict['data']['_id'], 
+  habitica_user_table.insert({'user_id': our_uuid, 
+    'last_timestamp': arrow.utcnow().timestamp,
```

Comment:
You might want to reuse the same technique so that I can roll back and re-run the pipeline in a unified fashion if there are errors. But we need to think about what that means from a habitica perspective.